lma-meetingassist-setup-stack/qna-ma-healthcare-demo.jsonl

```diff
@@ -0,0 +1,7 @@
+{"qid":"CustomNoMatches","a":"Default message - you'll see this only if the lambda hook failed","type":"qna","q":["no_hits"],"l":"<QNAMeetingAssistLambdaHookFunctionName>","args":["{\"AnswerPrefix\":\"Assistant Answer:\", \"QueryPrefix\":\"Assistant Query:\", \"ShowContextText\":true, \"ShowSourceLinks\":true}"]}
+{"qid":"AA.SummarizeCall","a":"Default message - you'll see this only if the lambda hook failed","type":"qna","q":["Summarize"],"l":"<LCASummarizeCallFunctionName>","args":["{transcript}<br><br>Carefully read the above meeting transcript. It is a conversation of a clinician and patient. List the active participants, and then summarize the meeting in no more than 5 sentences, using the speakers names."],"r":{"buttons":[{"text":"Summarize","value":"QID::AA.SummarizeCall"},{"text":"Topic","value":"QID::AA.CurrentTopic"},{"text":"List Symptoms","value":"QID::AA. PatientSymptoms"},{"text":"Ask Assistant!","value":"QID::AA.AskAssistant"}],"title": "Buttons"}}
+{"qid":"AA.CurrentTopic","a":"Default message - you'll see this only if the lambda hook failed","type":"qna","q":["Topic"],"l":"<LCASummarizeCallFunctionName>","args":["{transcript}<br><br>Identify the most recent issue or topic discussed in this meeting."], "r":{"buttons":[{"text":"Summarize","value":"QID::AA.SummarizeCall"},{"text":"Topic","value":"QID::AA.CurrentTopic"},{"text":"List Symptoms","value":"QID::AA. PatientSymptoms"},{"text":"Ask Assistant!","value":"QID::AA.AskAssistant"}],"title": "Buttons"}}
+{"qid":"AA. PatientSymptoms","a":"Default message - you'll see this only if the lambda hook failed","type":"qna","q":["Identify actions"],"l":"<LCASummarizeCallFunctionName>","args":["{transcript}<br><br>Please review the above medical transcript and extract all the symptoms mentioned by the patient into a bulleted list. Present the symptoms in a clear, organized, and easy-to-read format with each symptom on a new line. If a symptom is mentioned multiple times, include it only once in the list. If there are any abbreviations or medical jargon that may be unclear to a layperson, please spell out the full term. In case of any ambiguity or if you need additional context to accurately identify the symptoms, feel free to ask for clarification. Example output: - Patient report headache with nausea during the conversation- Difficulty of breathing was also highlighted with chest pain(angina)"],"r":{"buttons":[{"text":"Summarize","value":"QID::AA.SummarizeCall"},{"text":"Topic","value":"QID::AA.CurrentTopic"},{"text":"List Symptoms","value":"QID::AA. PatientSymptoms"},{"text":"Ask Assistant!","value":"QID::AA.AskAssistant"}],"title": "Buttons"}}
```

Comment:
Although this items is for 'Patient Symptoms' the default "question" in the item remains "List actions" which looks like an artifact inherited from the original item to edited from regular LMA..  
Best to change it to something relevant, like 'List patient symptoms'.
![image](https://github.com/aws-samples/amazon-transcribe-live-meeting-assistant/assets/10953374/a188e4ee-c504-4fc5-aca8-89479623ef70)

Reply:
Fixed in the latest push.